docs/docs/quick-start.md

```diff
@@ -1,22 +1,208 @@
+
+Now the Atlas cluster and Vector Search index are ready for you to use in your app.
+
+For more information on setting up MongoDB for the Chatbot Framework,
+refer to the [MongoDB & Atlas Vector Search](./mongodb.md) guide.
+
+### 2. Set up the Project
+
+Clone the repository with the quick start source code:
+
+```shell
+git clone https://github.com/mongodb/chatbot.git
+```
+
+Enter the directory with the quick start source code:
+
+```shell
+cd examples/quick-start
+```
+
+All the remaining steps will be run from `examples/quick-start` directory.
+
+This directory contains three packages, located in the `examples/quick-start/packages`:
+
+- `ingest`: Contains implementation of the MongoDB Ingest CLI, which ingests the content that the chatbot uses to answer questions.
+- `server`: Contains implementation of the MongoDB Chatbot Server, which serves the chatbot API.
+- `ui`: Contains implementation of the MongoDB Chatbot UI, which provides a UI to query the chatbot server.
+
+Install the dependencies for all the packages:
+
+```shell
+npm install
+```
+
+:::note
+
+The quick start uses [npm workspaces](https://docs.npmjs.com/cli/v8/using-npm/workspaces)
+to manage the dependencies for all the packages.
+
+:::
+
+Create an `.env` file based on the .env.example file in the root of the project:
+
+```shell
+cp .env.example .env
+```
+
+In the `.env` file, fill in the values for `MONGODB_CONNECTION_URI` and `OPENAI_API_KEY`
+with you Atlas connection URI and OpenAI API Key, respectively.
+
+### 3. Ingest Content
+
+In this step, you will ingest the content that the chatbot uses to answer questions
+into the `embedded_content` collection indexed with Atlas Vector Search.
+
+From the root of the project, run:
+
+```shell
+npm run ingest:all
+```
+
+If you've run the command successfully, you should an output resembling the following
+in your terminal:
+
+```shell
+{"level":"info","message":"Logger created"}
+{"level":"info","message":"Last successful run date: Thu Jan 04 2024 12:23:27 GMT-0500 (Eastern Standard Time)"}
+{"level":"info","message":"Loaded sources:\n- mongodb-rag-framework"}
+{"level":"info","message":"Fetching pages for mongodb-rag-framework"}
+{"level":"info","message":"Created /var/folders/v3/128h981j6vx4ncq_68qcg2cm0000gp/T/mongodb-rag-frameworkV3BE8d for https://github.com/mongodb/chatbot/"}
+# ...
+{"level":"info","message":"Updating last successful run date"}
+```
+
+To learn more about how you can configure the MongoDB Ingest CLI,
+refer to the [Configure the Ingest CLI](./ingest/configure.md) guide.
+
+### 4. Query the Chatbot
+
+In this step, you will spin up a server and frontend to query the chatbot.
+You'll be able to ask questions about the MongoDB Chatbot Framework
+using the data you ingested in the previous step.
+
+Start the chatbot server and UI with:
+
+```shell
+npm run dev
+```
+
+Open http://localhost:5173/ in your browser to see the UI.
```

Comment:
Dumb question but just checking that because one is using their own OpenAI API key, there's no need to worry about VPN/MongoDB/Azure access rules etc?

Reply:
nah, there's none of that access control on the openai version.